doc/graphl/querysamples/Sample_ParentChildrenSiblings.md

```diff
@@ -0,0 +1,79 @@
+## Get objects Parent/Children/Siblings
+
+For example, to get `berlina` object's (id:261) parent, children and siblings 
+
+![](../../img/graphql/sample_parentChildrenSiblings.png)
+
+### Request
+
+```graphql
+{
+  getCar(id: 261) {
+    id
+    name
+    
+    parent {
+      ... on object_Car {
+        id
+        name
+      }
+    }
+    
+    children {
```

Comment:
please mention the possibility to filter the type (folder, variant, ...) as there was already a question about that